src/config.rs

```diff
@@ -128,6 +128,8 @@ configuration_option_enum! { WriteMode:
     Plain,
     // Outputs a checkstyle XML file.
     Checkstyle,
+    // Output the changed lines
+    Modified,
```

Comment:
Could you note that this option is only meant to be used via an API

Reply:
Ok.